apps/sms/views/conversation/js/conversation.js

```diff
@@ -79,6 +84,13 @@ var ConversationView = {
     preventLastCall: false
   },
 
+  LATE_NOTICE_OPTS: {
```

Comment:
nit: instead of OPTS you can use LIMITS; and then you can remove _LIMIT from all properties :)